test/onnx/test_onnx_export.py

```diff
@@ -0,0 +1,62 @@
+# Owner(s): ["module: onnx"]
```

Comment:
I'm OK with this for now but I'll want to combine it with this file once it's merged:
https://github.com/pytorch/pytorch/pull/73284/files#diff-630243becc710c122152dc52e75109e86fc92a505fb293584e47ad3e4031fb65